### PR TITLE
Location update script

### PR DESCRIPTION
**Pull request by patrickdent**

### CODE REVIEW

File: app/views/authors/show.html.erb [hunk 1]
```diff
@@ -16,9 +16,11 @@
 
   <tbody>
     <% @author.books.each do |b| %>
-      <% on_loan = "on-loan" if !b.available %>
+      <% missing = "missing" if b.missing %>
+      <% on_loan = "on-loan" unless b.available || missing %>
```
Comment (cherimarie) [inline app/views/authors/show.html.erb:20]: I think this should be b.missing

File: app/views/books/index.html.erb [hunk 2]
```diff
@@ -14,9 +14,11 @@
   </thead>
   <tbody>
     <% @books.each do |b| %>
-      <% on_loan = "on-loan" if !b.available %>
+      <% missing = "missing" if b.missing %>
+      <% on_loan = "on-loan" unless b.available || missing %>
```
Comment (cherimarie) [inline app/views/books/index.html.erb:18]: I think this should be b.missing

File: app/views/books/show_list.html.erb [hunk 3]
```diff
@@ -13,9 +13,11 @@
   </thead>
   <tbody>
     <% @books.each do |b| %>
-      <% on_loan = "on-loan" if !b.available %>
+      <% missing = "missing" if b.missing %>
+      <% on_loan = "on-loan" unless b.available || missing %>
```
Comment (cherimarie) [inline app/views/books/show_list.html.erb:17]: again

File: app/views/genres/show.html.erb [hunk 4]
```diff
@@ -16,9 +16,11 @@
 
   <tbody>
     <% @genre.books.each do |b| %>
-      <% on_loan = "on-loan" if !b.available %>
+      <% missing = "missing" if b.missing %>
+      <% on_loan = "on-loan" unless b.available || missing %>
```
Comment (cherimarie) [inline app/views/genres/show.html.erb:20]: ditto

**Comment by patrickdent** [comment]
missing is declared in the line above each of these - this way missing will override on_loan

**Pull request merged**